Gems/Atom/RHI/Code/Source/RHI/RayTracingAccelerationStructure.cpp

```diff
@@ -88,6 +88,13 @@ namespace AZ
             return this;
         }
 
+        RayTracingTlasDescriptor* RayTracingTlasDescriptor::Transparent(bool transparent)
```

Comment:
Minor: Function name feels a bit weird. A better name would be SetTransparent.

Reply:
The name is appropriate since the functions in this class are used in a pipeline fashion, like so:
                            ->HitGroupIndex(0)
                            ->Blas(subMesh.m_blas)
                            ->Transform(subMesh.m_mesh->m_transform)
                            ->NonUniformScale(subMesh.m_mesh->m_nonUniformScale)
                            ->Transparent(subMesh.m_irradianceColor.GetA() < 1.0f)